chroma-agent/chroma_agent/device_plugins/linux_components/block_devices.py

```diff
@@ -94,7 +95,13 @@ def as_device(x):
         'device_path': x.get('DEVPATH'),
         'partition_number': x.get('ID_PART_ENTRY_NUMBER'),
         'is_ro': x.get('IML_IS_RO'),
-        'parent': None
+        'parent': None,
+        'dm_multipath': x.get('DM_MULTIPATH_DEVICE_PATH'),
+        'dm_lv': x.get('DM_LV_NAME'),
+        'dm_vg': x.get('DM_VG_NAME'),
+        'dm_uuid': x.get('DM_UUID'),
```

Comment:
Could this be None?

Reply:
when the device is not related to a device mapper